docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -502,16 +503,11 @@ The flags can also appear anywhere later on the command line.
 The acceptable flags include all those of `kubectl` except for
 `--context`.  This command also accepts the following flags.
 
-- `--imw workspace_path`: specifies which workspace to use as the
-  inventory management workspace.  The default value is the current
-  workspace.
+- `--imw space_name`: specifies which space to use as the
+  inventory management workspace.  
 - `-X`: turn on debug echoing of the commands inside the script that
   implements this command.
 
```

Comment:
The unusual environment variable sensitivities (SM_CONFIG, SM_CONTEXT) need to be documented.